.github/workflows/apkPipeline.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - 'release/**'
       - 'main'
+      - 'bugfix/**'
```

Comment:
This will be deleted before merging. This is just to make sure that the apk is generated correctly 
```suggestion
```